translate.json

```diff
@@ -2,297 +2,556 @@
+            "DE":"<b>RMU-40 S2</b>"
         },"rmu_s3": {
             "SE":"<b>RMU-40 S3</b>",
-            "EN":"<b>RMU-40 S3</b>"
+            "EN":"<b>RMU-40 S3</b>",
+            "DE":"<b>RMU-40 S3</b>"
         },"samba_stop": {
-            "SE":"<b>Stoppa Samba</b>"
+            "SE":"<b>Stoppa Samba</b>",
+            "EN":"<b>Stop Samba</b>",
+            "DE":"<b>Samba Stoppen</b>"
         },"samba_start": {
-            "SE":"<b>Starta Samba</b>"
+            "SE":"<b>Starta Samba</b>",
+            "EN":"<b>Start Samba</b>",
+            "DE":"<b>Samba starten</b>"
         },"nodered_restart": {
-            "SE":"<b>Starta om Node-RED</b>"
+            "SE":"<b>Starta om Node-RED</b>",
+            "EN":"<b>Restart Node-RED</b>",
+            "DE":"<b>Node-RED neu starten</b>"
         },"nodered_restart_info": {
-            "SE":"<p><b>Startar om Node-RED</b></p>"
+            "SE":"<p><b>Startar om Node-RED</b></p>",
+            "EN":"<p><b>Trigger a restart of Node-RED.</b></p>",
+            "DE":"<p><b>Neustart von Node-RED auslösen.</b></p>"
         },"hw_restart": {
-            "SE":"<b>Starta om hårdvaran</b>"
+            "SE":"<b>Starta om hårdvaran</b>",
+            "EN":"<b>Restart NibePi hardware</b>",
+            "DE":"<b>NibePi hardware neu starten</b>"
         },"hw_restart_info": {
-            "SE":"<p><b>Startar om NibePi hårdvara</b></p>"
+            "SE":"<p><b>Startar om NibePi hårdvara</b></p>",
+            "EN":"<p><b>Triggers a restart of the NibePi hardware.</b></p>",
+            "DE":"<p><b>Neustart der NibePi Hardware auslösen.</b></p>"
         },"hw_shutdown": {
-            "SE":"<b>Stäng av NibePi hårdvaran</b>"
+            "SE":"<b>Stäng av NibePi hårdvaran</b>",
+            "EN":"<b>Shutdown NibePi</b>",
+            "DE":"<b>NibePi herunterfahren</b>"
         },"hw_shutdown_info": {
-            "SE":"<p><b>Stänger av NibePi hårdvaran</b></p>"
+            "SE":"<p><b>Stänger av NibePi hårdvaran</b></p>",
+            "EN":"<p><b>Triggers a shutdown of the NibePi hardware.</b></p>",
+            "DE":"<p><b>Herunterfahren der NibePi Hardware auslösen.</b></p>"
         },"mqtt_info": {
-            "SE":"<p><b>MQTT</b></p><br>Aktivera åtkomst till MQTT broker.<br>NibePi har en inbyggd broker på 127.0.0.1:1883<br>Välj annars en extern broker.<br>Aktivera HA Discovery om Home Assistant automatiskt ska upptäcka aktiverade givare från NibePi"
+            "SE":"<p><b>MQTT</b></p><br>Aktivera åtkomst till MQTT broker.<br>NibePi har en inbyggd broker på 127.0.0.1:1883<br>Välj annars en extern broker.<br>Aktivera HA Discovery om Home Assistant automatiskt ska upptäcka aktiverade givare från NibePi",
+            "EN":"<p><b>MQTT</b></p><br>Activate the MQTT broker.<br>NibePi has an internal MQTT broker that runs on 127.0.0.1:1883<br>If deactivated, you can use an external broker.<br>Activate HA auto discovery, if HomeAssistant should detect NibePi's sensor automatically.",

```

Comment:
"<br>If deactivated, you can use an external broker.<br>Activate HA auto discovery, if HomeAssistant should detect NibePi's sensor automatically.","
For external broker, it needs to be activated too. The swedish text is a bit unclear so I will update that.
I would say"<br>You can also use an external broker just by changing the connection details. Activate HA auto discovery, if Home Assistant should detect NibePi's sensors automatically.","

Reply:
Sure, sounds good.
Can you check the electricity price control strings also. Not sure if I got everything right from machine translation.
Also the calibration for the auto hot water luxury mode. I'm not 100% positive I got everything right.

Reply:
Nice work, looks very well! I just changed some minor typos and one misunderstanding in the airflow, degree minutes section.
If you can double check the deutch translation on the things i changed it will be ready.
Very nice work!!

Reply:
Can you shorten these lines? Just by 2 words or more. It dosent fit the frontend size.
"Stelle hier den Sollwert für die Raumtemperatur ein. Der selbe Wert wird in der Wärmepump"
"Lege fest, wie start der Raumtemperatursensor die Vorlauftemperatur beeinflusst." (One word shorter is enough)
I can see the english version of these sentences has the same problem.